django_react_proj/users-fe/src/components/UserList/UserList.js

```diff
@@ -32,7 +32,9 @@ const Actions = ({ user, resetState }) => {
   return (
     <>
       <TaskButton setDoneState={setDoneState} />
+      <> </>
```

Comment:
What does it to?

Reply:
It creates mini space between each button